includes/class-signature.php

```diff
@@ -206,7 +206,8 @@ public static function maybe_double_knock( $response, $parsed_args, $url ) {
 		// Remove this filter to prevent infinite recursion.
 		\remove_filter( 'http_response', array( self::class, 'maybe_double_knock' ) );
 
-		if ( 401 === wp_remote_retrieve_response_code( $response ) ) {
+		$response_code = \wp_remote_retrieve_response_code( $response );
+		if ( 400 <= $response_code && 500 > $response_code ) {
```

Comment:
we can do without yoda condition in greater/less than comparisons? this is really hard to read!